guides/micronaut-mongodb-synchronous/micronaut-mongodb-synchronous.adoc

```diff
@@ -0,0 +1,167 @@
+include::{commondir}/common-header.adoc[]
+
+You will use https://www.mongodb.com[MongoDB] for persistence.
+
+include::{commondir}/common-requirements.adoc[]
+
+include::{commondir}/common-completesolution.adoc[]
+
+== Writing the App
+
+include::{commondir}/common-cli-or-launch.adoc[]
+
+[source,bash]
+----
+mn create-app \
+    --features=mongo-sync,graalvm example.micronaut.micronautguide \
+    --build=@build@ \
+    --lang=@lang@
+----
+
+include::{commondir}/common-build-lang-arguments.adoc[]
+
+If you use Micronaut Launch, select Micronaut Application as application type and add the `mongo-sync` feature.
+
+include::{commondir}/common-default-package.adoc[]
+
+diffLink:[]
+
+=== POJO
+
+Create `Fruit` POJO:
+
+source:Fruit[]
+
+callout:introspected[1]
+callout:constraints[2]
+
+=== Repository
+
+Create an interface to encapsulate the CRUD actions for `Fruit`. 
+
+source:FruitRepository[]
+
+callout:valid[1]
+
+=== Controller
+
+Create `FruitController`:
+
+source:FruitController[]
+
+callout:controller[number=1,arg0=/fruits]
+callout:constructor-di[number=2,arg0=FruitRepository]
+callout:executes-on[3]
+callout:get[number=4,arg0=list,arg1=/fruits]
+callout:post[number=5,arg0=save,arg1=/fruits]
+callout:at-status[6]
+callout:valid[7]
+
+=== Configuration
+
+Create a configuration object to encapsulate the MongoDB database's name and collection's name.
```

Comment:
I'd remove the `'s` from database and collection. Both are correct but it reads better without.

Reply:
I've remove the `'s`